### DISCUSSION TITLE
Gmail New UI Icons are missing on CefSharp Browser

### DISCUSSION THREAD

**Discussion by mukeshmishra996** [cefsharp/CefSharp#4237] [General]
Hi Team,
When I am trying to load Gmail inside the CefSharp Browser and switch to new Gmail UI. Some of the Gmail menu icons are missing. When I opened dev tool and verify, Gmail new UI using CSS mask property to load icons. Can you please help us to fix this issue. Please find below details for your reference:
User-Agent: Mozilla Firefox
Code Snipping:
`-webkit-mask: url(https://www.gstatic.com/images/icons/material/system_gm/2x/star_border_black_20dp.png) center/contain;`

 `mask: url(https://www.gstatic.com/images/icons/material/system_gm/2x/star_border_black_20dp.png) center/contain;`

![MicrosoftTeams-image](https://user-images.githubusercontent.com/14114368/188665887-bd390760-0b6e-4aeb-98ab-645fde5292a1.png)

**Comment by amaitland** [comment]
Already asked at https://stackoverflow.com/questions/73580207/mask-property-of-css-isnt-working-in-cefsharp-browser
